tests/libvmi/vmi.go

```diff
@@ -71,6 +71,14 @@ func WithResourceMemory(value string) Option {
 	}
 }
 
+// WithAlternativeNamespace uses tests.NamespaceTestAlternative as the vmi
```

Comment:
The doc string does not match the function name nor the implementation

Reply:
Done